changelogs/unreleased/add-ubuntu-noble.md

```diff
@@ -0,0 +1,3 @@
+## bugfix/build
+
+* Deploy packages for Ubuntu Noble (24.04).
```

Comment:
```suggestion
* Deployed packages for Ubuntu Noble (24.04).
```

Reply:
These packages are not deployed by this commit, but a logic that performs the deploy for Ubuntu Noble is added. So, the future packages will be deployed for the new Ubuntu version too.

It seems, I can't just say 'deployed' here.

Reply:
I can say 'Ubuntu Noble (24.04) is now supported' without any details about build/testing/deployment.

Reply:
Changed this way.